Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -300,6 +315,49 @@ public void Download_Error()
 
                 var lastProgress = progressList.LastOrDefault();
                 Assert.That(lastProgress.Status, Is.EqualTo(DownloadStatus.Failed));
+                GoogleApiException exception = (GoogleApiException) lastProgress.Exception;
+                Assert.That(exception.HttpStatusCode, Is.EqualTo(handler.StatusCode));
+                // Just a smattering of checks - if these two pass, it's surely okay.
+                Assert.That(exception.Error.Code, Is.EqualTo(error.Code));
+                Assert.That(exception.Error.Errors[0].Message, Is.EqualTo(error.Errors[0].Message));
+            }
+        }
+
+        [Test]
+        public void Download_Error_PlaintextResponse()
+        {
+            var downloadUri = "http://www.sample.com";
+            var chunkSize = 100;
+            var responseText = "Not Found";
+
+            var handler = new MultipleChunksMessageHandler { ErrorResponse = responseText };
+            handler.StatusCode = HttpStatusCode.NotFound;
+            handler.ChunkSize = chunkSize;
+            // The media downloader adds the parameter...

```

Comment:
The `handler.DownloadUri` specifies the URI we expect to be called. We provide `downloadUri` to `MediaDownloader`, but it appends `alt=media` - so I'm saying that we expect that.

(In reality, if you use MediaDownloader with a MediaLink from a GCS element, we *do* get alt=media twice, which is weird.)

This is basically the equivalent of line 213 earlier, but we don't need to be conditional.